afollestad/material-dialogs
# Revert changes to accessing color resources

Issue
----

#729 

Solution
----

Manually switch off SDK version, to call deprecated `context.getResources().getColor( )` method. See `DialogUtils.getColor( )`. This means we no longer need `ContextCompat.getColor( )`.

Test
----

* Create a new (app) project that has first-level dependency on `com.android.support:appcompat-v7:22.2.1` and 'core-release.aar' from this branch 
* Add a single `MaterialDialog`
* Build project
* Ensure no error: 

```
java.lang.NoSuchMethodError: No static method getColor(Landroid/content/Context;I)I in class Landroid/support/v4/content/ContextCompat; or its super classes (declaration of 'android.support.v4.content.ContextCompat'
```
